pytket/pytket/__init__.py

```diff
@@ -26,3 +26,4 @@
 import pytket.telemetry
 
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
+__version__ = _dunamai.Version.from_git().base
```

Comment:
Surely this will depend on the directory the user is in when they import pytket?

Reply:
Well, the user has to be where the VCS can be queried. Maybe best to have it static ?

Reply:
But a typical pytket user has not cloned the git repo, they just pip install the package.

Reply:
I guess you can integrate it when you build the wheel, from the [docs](https://pypi.org/project/dunamai/):

```
from setuptools import setup
from dunamai import Version

setup(
    name="your-library",
    version=Version.from_any_vcs().serialize(),
)
```

Reply:
Dunamai exists as a [GitHub Actions](https://github.com/marketplace/actions/run-dunamai). Might want that for every release ?

Reply:
We can set up a workflow that sets the version statically at every release. Might do the trick.

Reply:
I think I prefer the approach of adding it in `setup.py` (or `pyproject.toml`?), replacing the current use of `setuptools_scm`, but we should thoroughly test it.

Reply:
Yeah, can do in `pyproject.toml` too I think. Well, either one or the other unless there is a preference for the later.

Reply:
As we will soon be moving away from `setup.py` I would do it in `pyproject.toml` if possible (that is where `setuptools_scm` is at the moment).

Reply:
OK got that. I wasn't sure about the status of that transition yet. Will do.